src/makeRobot.js

```diff
@@ -39,6 +39,74 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
```

Comment:
Мне вчера подсказали, что тут можно использовать упрощенную запись, если ключ равен значению (name: name, wheels: wheels), и все будет так же работать:
```suggestion
  const robot = {
    name,
    wheels,
    version,
    coords: {
      x: 0,
      y: 0,
    },
```

Reply:
Дякую за підказку, тепер теж буду знати)